boards/arm/thingy91_nrf9160/thingy91_nrf9160_ns_defconfig

```diff
@@ -8,6 +8,10 @@ CONFIG_ARM_MPU=y
 # Enable TrustZone-M
 CONFIG_ARM_TRUSTZONE_M=y
 
+# Use SPM instead of TF-M to be backwards-compatible with MCUboot from NCS v1.9.1 and earlier.
```

Comment:
https://github.com/nrfconnect/sdk-nrf/blame/5454e1a393205894e9f8319438374cafa2a573b4/boards/arm/thingy91_nrf9160/Kconfig.defconfig#L12L26
Removing these lines was the change I expected.

This works too though, but may be more confusing to figure out what is happening.